src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -379,7 +379,8 @@ object ErrorAggregator {
         .repartition(opts.numParquetFiles())
         .write
         .mode("overwrite")
-        .parquet(s"${outputPath}/${outputPrefix}/submission_date_s3=${currentDate.toString(dateFormat)}")
+        .partitionBy("submission_date_s3")
+        .parquet(s"${outputPath}/${outputPrefix}")
```

Comment:
This will overwrite everything in `{outputPath}/${outputPrefix}`, deleting old data. Unless we use the new write mode, the `overwrite` mode needs to be used on the specific partition.

Reply:
Yes, I hardcoded it to use new write mode in https://github.com/mozilla/telemetry-streaming/pull/124/files#diff-e9e9e1ae5916120d8ace633ccc23891cR404
There's also a check that'll fail the job on previous Spark version (since the dynamic mode wasn't there yet): https://github.com/mozilla/telemetry-streaming/pull/124/files#diff-e9e9e1ae5916120d8ace633ccc23891cR407

Reply:
Ah great, I totally missed that.